config/software/binutils.rb

```diff
@@ -17,7 +17,8 @@
 name "binutils"
 default_version "2.26"
 
-version("2.26") { source md5: "9615feddaeedc214d1a1ecd77b6697449c952eab69d79ab2125ea050e944bcc1" }
+version("2.35") { source sha256: "a3ac62bae4f339855b5449cfa9b49df90c635adbd67ecb8a0e7f3ae86a058da6" }
+version("2.26") { source sha256: "9615feddaeedc214d1a1ecd77b6697449c952eab69d79ab2125ea050e944bcc1" }
```

Comment:
```suggestion
version("2.26") { source md5: "9615feddaeedc214d1a1ecd77b6697449c952eab69d79ab2125ea050e944bcc1" }
```

Reply:
I have no idea how that was working before but it was not a md5 checksum before